packages/graphback-datasync/src/DatasyncPlugin.ts

```diff
@@ -56,32 +57,24 @@ export class DatasyncPlugin extends GraphbackPlugin {
         }
 
         models.forEach((model: ModelDefinition) => {
-            // TODO use `versioned` marker to check if we should add version
-            const modifiedType = schemaComposer.getOTC(model.graphqlType.name);
-            modifiedType.addFields({
-                version: 'Int'
-            });
-
-            // TODO: Add version to all input types
-            try {
-                const inputType = schemaComposer.getITC(getInputTypeName(model.graphqlType.name, GraphbackOperationType.FIND))
-                if (inputType) {
-                    inputType.addFields({
-                        version: 'Int'
-                    });
+            const SyncResultList = new GraphQLObjectType({
+                name: `${model.graphqlType.name}DeltaList`,
+                fields: {
+                    items: { type: GraphQLList(model.graphqlType)},
+                    lastSync: { type: GraphQLString}
                 }
```

Comment:
I am wondering if we should deal / or there are plans to deal with pagination too?

Reply:
Presently this setup returns all changed documents, but I think paginating it could be worthwhile, @wtrocki  what do you think?

Reply:
Offset-based pagination won't work well with data synchronisation I believe, since the result set is dynamic and can be very different after each request. 

We would first need to implement cursor-based pagination, which is probably outside the scope of this PR.

https://www.sitepoint.com/paginating-real-time-data-cursor-based-pagination/

Reply:
+1 with what @craicoverflow said. I am fine with the answers here.